views/layouts/main.hbs

```diff
@@ -54,10 +41,53 @@
     {{{body}}}
 
     <!-- Latest compiled and minified JavaScript -->
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <script src='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js' integrity='sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa' crossorigin='anonymous'></script>
 
-    <script type="text/javascript" src="/bower_components/bootstrap-validator/dist/validator.js"></script>
-    <script type="text/javascript" src="/javascripts/app.js"></script>
-  
+    <script type='text/javascript' src='/bower_components/bootstrap-validator/dist/validator.js'></script>
+    <script type='text/javascript' src='/javascripts/app.js'></script>
+    <script src='https://js.docusign.com/bundle.js'></script>
+    <script>
+        window.DocuSign.loadDocuSign('{{client_id}}')
+            .then((docusign) => {
+                var url = '{{signing_url}}'.replace('&#x3D;', '=');
+                const signing = docusign.signing({
+                    url: url,
+                    displayFormat: 'focused',
+                    style: {
+                        /** High-level variables that mirror our existing branding APIs. Reusing the branding name here for familiarity. */
```

Comment:
Not sure if this is new code, but this comment says "branding APIs" which is not a thing. We have in the eSignature API Branding capabilities, may want to reword here